src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -37,6 +37,16 @@ public AddressBook(ReadOnlyAddressBook toBeCopied) {
         resetData(toBeCopied);
     }
 
+    /**
+     * Returns the current size of the internalList in the UniquePersonList variable
+     * persons
```

Comment:
Consider add one more empty line here to separate the first sentence from @return sentence.